crypto/blake2/build.info

```diff
@@ -0,0 +1,3 @@
+LIBS=../../libcrypto
+SOURCE[../../libcrypto]=\
+        blake2b.c blake2s.c {- $target{blake2b_obj} -}
```

Comment:
`$target{blak2_obj}` is fine for the older `Makefile.in`. The new `build.info` files deal with source, and therefore use the names from `Configurations/00-base-templates.conf` directly. That's where the asm variable comes from. 

I noticed you removed it entirely, which is technically fine now, but will not be fine as soon as asm source is added.